docs/graphql.md

```diff
@@ -1,115 +0,0 @@
-
```

Comment:
I don't currently use GraphQL, but since GraphQL used to be supported, I wonder if there is some benefit to keeping this page around, and just mentioning that there used to be support for it but no longer. And also perhaps to provide links/examples as to how someone could add a graphql app using one of the external libraries. (maybe just linking out to the compatible external libraries?)

Reply:
Hmm... Honestly I think we should have a versioned docs. I'm going to propose that on our Gitter channel.

In any case, if the versioned docs idea doesn't succeed, then yes, I guess we can let this page to live and modify its content.

Reply:
There are some tutorials integrating with Starlette already:
- [Ariadne - Starlette integration](https://ariadnegraphql.org/docs/starlette-integration.html)
- [tartiflette-asgi - Starlette integration](https://tartiflette.github.io/tartiflette-asgi/usage/#starlette)
- [starlette-graphene3 readme](https://github.com/ciscorn/starlette-graphene3)

I've also asked on Strawberry for a tutorial integration on their end:
- https://github.com/strawberry-graphql/strawberry/issues/1284

I'm thinking about saying a few words about the removal, and forwarding people to those tutorials.